src/host/cmdline.cpp

```diff
@@ -795,14 +791,11 @@ til::point CommandLine::_moveCursorRightByWord(COOKED_READ_DATA& cookedReadData)
             }
 
             // Skip the space block.
-            if (NextWord < BufLast && *NextWord == L' ')
+            for (; NextWord < BufLast; NextWord++)
             {
-                while (++NextWord < BufLast)
+                if (*NextWord != L' ')

```

Comment:
Can this be simplified to...

```c++
for(; NextWord < BufLast && *NextWord == L' '; ++NextWord) { }
```

?

Maybe it is in violation of the "more clever" message I sent above.